index.js

```diff
@@ -11,11 +11,14 @@ if (process.env.SENTRY_DSN) {
     allowUrls: [
       /^https:\/\/ecobalyse\.beta\.gouv\.fr/,
       /^https:\/\/staging-ecobalyse\.incubateur\.net/,
+      // Review apps
+      /^https:\/\/ecobalyse-pr.*\.osc-fr1\.scalingo\.io/,
     ],
     ignoreErrors: [
       // Most often due to DOM-aggressive browser extensions
       /_VirtualDom_applyPatch/,
     ],
+    environment: process.env.IS_REVIEW_APP ? "review-app" : process.env.NODE_ENV || "development",
```

Comment:
Ideally we should never collect events on `development` environment, don't you think? How could we proceed to ensure we're only collecting them on `production`, and maybe eventually on `staging`?

Reply:
As localhost is not in the allowedUrls we should not collect dev errors. But… `localhost` seems to be enabled on `sentry.incubateur.net`
![image](https://github.com/user-attachments/assets/8edddd24-b765-4aa1-8d68-f8f9b74f1d1f)

For the server side sentry, we could init it only if `NODE_ENV` is `production`, what do you think?

Reply:
Sounds good to me!